### PR TITLE
fix: use the right lettercase, add npm start

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 .DS_Store
+package-lock.json
```
Comment (frederickfogerty) [inline .gitignore]: `package-lock.json` should not be ignored, according to https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

**Comment by frederickfogerty** [comment]
Hey @DanielRuf 👋 

Thanks for the PR! I support the use of npm scripts so this PR is very welcome. After resolving my review comment, I'm happy to merge.

File: .gitignore (hunk 1 above)
Comment (DanielRuf) [reply .gitignore]: Sure, but so far it is not part of the repo and I guess you (the team) ignored it globally or do not want to add the lockfile (which woul require manual dependency upgrades then).

Comment (frederickfogerty) [reply .gitignore]: We're keen to check it in from now on 👍

**Comment by frederickfogerty** [comment]
Thanks for making those changes @DanielRuf.

**Pull request merged**